src/core/server/saved_objects/migrations/state.ts

```diff
@@ -122,6 +122,10 @@ export interface BaseState extends ControlState {
     string,
     SavedObjectTypeExcludeFromUpgradeFilterHook
   >;
+  /**
+   * DocLinks for savedObjects. to reference online documentation
+   */
+  readonly migrationDocLinks: Record<string, string>;
```

Comment:
Adding the documentation links to state was the easiest way to make them available to all the actions within the migration algorithm.